accessibilityTest/ContrastChecker.ts

```diff
@@ -11,12 +11,16 @@ export interface IColor {
 export class ContrastChecker {
   public static readonly MinimumContrastRatio = 3;
 
-  public static getContrastWithBackground(element: HTMLElement, colorAttributeName: keyof CSSStyleDeclaration = 'color') {
+  public static getContrastWithBackground(
+    element: HTMLElement,
+    colorAttributeName: keyof CSSStyleDeclaration = 'color',
+    backgroundElement: HTMLElement = element
```

Comment:
This optional parameter was added because the settings button was checking its border contrast against its own background, which wasn't desirable for this test case.